packages/web-ui/src/providers/commitProvider.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import { createContext, ReactNode, useContext } from 'react'
+
+import { HEAD_COMMIT } from '@latitude-data/core/browser'
+
+// const HEAD_COMMIT = 'HEAD'
+
+interface CommitContextType {
+  commitUuid: string | typeof HEAD_COMMIT
```

Comment:
Move this type to `core/src/constants` It will be handy for more cases. Also call the type somethiing like `HeadCommit`